TrafficCapture/captureOffloader/src/test/java/org/opensearch/migrations/trafficcapture/StreamChannelConnectionCaptureSerializerTest.java

```diff
@@ -370,18 +590,18 @@ protected CompletableFuture<Void> kickoffCloseStream(CodedOutputStreamHolder out
                     "Unknown outputStreamHolder sent back to StreamManager: " + outputStreamHolder);
             }
             var osh = (CodedOutputStreamAndByteBufferWrapper) outputStreamHolder;
-            log.trace("Getting ready to flush for " + osh);
-            log.trace("Bytes written so far... " + StandardCharsets.UTF_8.decode(osh.getByteBuffer().duplicate()));
+            log.atTrace().log(() -> "Getting ready to flush for " + osh);
```

Comment:
oh darn - all of those setMessage() calls that I've been doing should probably be like this